index.html

```diff
@@ -6,9 +6,38 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Homework 4</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./Styles/all.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
+    <link rel="stylesheet" href="./Styles/style.css">
 </head>
 <body>
+    <div class="container main">
+            <div class="leftSide">
```

Comment:
use common type for classes, if you wrote `leftSide`, then all your classes should be like `statNumber`

but better to use `left-side,` `stat-number`